### PR TITLE
Test calendarview

### PR DESCRIPTION
**Pull request by Gomez**

### CODE REVIEW

File: js/tests/views_calendarview_spec.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+	describe('CalendarView', function () {
+
+		var calendarview;
+
+		beforeEach(function () {
+			calendarview = new CalendarView({});
+		});
+
+		describe('Rendering', function () {
```
Comment (ChristophWurst) [inline js/tests/views_calendarview_spec.js]: do we really need the second `describe` block? I guess one *describe* and one *it* should be enough :wink:

**Comment by ChristophWurst** [comment]
test coverage decreases as more modules are loaded, but that's okay I guess

File: js/tests/views_calendarview_spec.js (hunk 1 above)
Comment (Gomez) [inline js/tests/views_calendarview_spec.js]: Ok, removed

**Comment by ChristophWurst** [comment]
:+1:

**Pull request merged**